packages/config/maintenance/lintConfigFiles.ts

```diff
@@ -861,20 +862,31 @@ Consider converting this parameter to unsigned using ${white(
 				...conditionalConfig.paramInformation.entries(),
 			].filter(([k]) => !!k.valueBitMask);
 
-			// Check if there are parameters with a single bit mask
 			const partialParamCounts = partialParams
 				.map(([k]) => k)
 				.reduce((map, key) => {
 					if (!map.has(key.parameter)) map.set(key.parameter, 0);
 					map.set(key.parameter, map.get(key.parameter)! + 1);
 					return map;
 				}, new Map<number, number>());
-			for (const [param, count] of partialParamCounts.entries()) {
-				if (count === 1) {
-					addError(
-						file,
-						`Parameter #${param} has a single bit mask defined. Either add more, or delete the bit mask.`,
-					);
+
+			for (const [key, paramInfos] of partialParams) {
+				if (partialParamCounts.get(key.parameter) == 1) {
+					for (const param of paramInfos) {
+						const bitMask = key.valueBitMask!;
+						const shiftAmount = getMinimumShiftForBitMask(bitMask);
+						const bitMaskWidth = getBitMaskWidth(bitMask);
+
+						if (
+							shiftAmount == 0 &&
+							param.valueSize >= bitMaskWidth / 8
```

Comment:
That's not correct IMO. You want to catch bitmasks that span the entire value, not part of it starting with the LSB.
```suggestion
							shiftAmount === 0 &&
							param.valueSize === bitMaskWidth / 8
```